event-details/attendees-list/AttendeesListView.tsx

```diff
@@ -0,0 +1,47 @@
+import { Headline } from "@components/Text"
+import { EventAttendee } from "@shared-models/Event"
+import React from "react"
+import { FlatList, StyleProp, View, ViewStyle } from "react-native"
+import { useAttendeesList } from "./AttendeesList"
+
+export type AttendeesListViewProps = {
+  renderAttendee: (eventAttendee: EventAttendee) => JSX.Element
+  ListHeaderComponent?: JSX.Element
+  style?: StyleProp<ViewStyle>
+} & Extract<ReturnType<typeof useAttendeesList>, { status: "success" }>
+
+export const AttendeesListView = ({
```

Comment:
Migrate base view (this one) over to AttendeesList.tsx, any extra views can be in different files